production/ksonnet/promtail/promtail.libsonnet

```diff
@@ -122,24 +122,16 @@ k {
             target_label: '__host__',
           },
 
-          // Drop pods without a app label
-          {
-            source_labels: ['__meta_kubernetes_pod_label_app'],
-            action: 'drop',
-            regex: '^$',
-          },
-
-          // Drop pods with a 'name' and an 'app' label.  They will have already been added by 
-          // the scrape_config that matches on the 'name' label
+          // Drop pods with a name label
           {
             source_labels: ['__meta_kubernetes_pod_label_name'],
             action: 'drop',
             regex: '.+',
           },
```

Comment:
Lets have this match the config for helm - either drop pods with name label, of keep pods without.